GSOC18/GeometryUtilities-master/interpolation/hexCells_to_squareCell_interpolation.py

```diff
@@ -304,3 +331,100 @@ def plot_hex_to_square_map(coef,hex_cells_dict,sq_cells_dict):
         #ax1.set_ylim(-160, 160)
         #ax1.set_aspect(1)
         plt.show()
+
```

Comment:
This function is separate and is meant for the "image" creation from the recorded hits in the dataframe.